Unity 2017.1.3 don't support GetFrameworksBuildPhaseByTarget yet

<!--
 1. IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MAY CLOSE YOUR ISSUE WITHOUT INVESTIGATION
 2. SEARCH EXISTING ISSUES FOR AN ANSWER: https://goo.gl/CBfk7K
 3. See our Common Issues documentation: https://goo.gl/T7siu5
 4. See our contributing guidelines: https://goo.gl/uNVgUv
-->

**Description:**
<!-- (write below this line) -->
I using unity 2017.1.3. When I switch to iOS platform this error come out:
```
Assets/OneSignal/Editor/PostProcessBuildPlayer_iOS.cs(156,49): error CS1061: 
Type `UnityEditor.iOS.Xcode.PBXProject' does not contain a definition for 
`GetFrameworksBuildPhaseByTarget' and no extension method `GetFrameworksBuildPhaseByTarget' 
of type `UnityEditor.iOS.Xcode.PBXProject' could be found. 
Are you missing an assembly reference?
```
In `PostProcessBuildPlayer_iOS.cs` I saw in line 126 `#if UNITY_2017_1_OR_NEWER && !UNITY_CLOUD_BUILD`
But when I research in Unity 2017.1 document It seem not support this function yet
https://docs.unity3d.com/2017.1/Documentation/ScriptReference/iOS.Xcode.PBXProject.html
Just from Unity 2017.2 It began support this function
https://docs.unity3d.com/2017.2/Documentation/ScriptReference/iOS.Xcode.PBXProject.html

Please update code or Correct me if I'm wrong.
Thank you very much.

**Environment**
<!-- Example:
1. What version of the Unity OneSignal SDK are you using?
2. Provide a list of your project dependencies
3. Does the issue occur only in iOS, Android, etc?
 -->
1. OneSignal version: Unity-SDK-2.7.4
2. Unity 2017.1.3f1
3. iOS


<!--
  SEARCH EXISTING ISSUES FOR AN ANSWER: https://goo.gl/CBfk7K
-->

--- a/OneSignalExample/Assets/OneSignal/Editor/PostProcessBuildPlayer_iOS.cs
+++ b/OneSignalExample/Assets/OneSignal/Editor/PostProcessBuildPlayer_iOS.cs
@@ -11,7 +11,7 @@
    using UnityEditor.iOS.Xcode;
    using System.Text;
 
-   #if UNITY_2017_1_OR_NEWER
+   #if UNITY_2017_2_OR_NEWER
       using UnityEditor.iOS.Xcode.Extensions;
    #endif
 
@@ -35,7 +35,7 @@
          // UserNotifications.framework is required by libOneSignal.a
          project.AddFrameworkToProject(targetGUID, "UserNotifications.framework", false);
 
-         #if UNITY_2017_1_OR_NEWER && !UNITY_CLOUD_BUILD
+         #if UNITY_2017_2_OR_NEWER && !UNITY_CLOUD_BUILD
 
             var extensionTargetName = "OneSignalNotificationServiceExtension";
             var pathToNotificationService = path + "/" + extensionTargetName;
@@ -123,7 +123,7 @@
          File.WriteAllText(projectPath, project.WriteToString());
       }
 
-      #if UNITY_2017_1_OR_NEWER && !UNITY_CLOUD_BUILD
+      #if UNITY_2017_2_OR_NEWER && !UNITY_CLOUD_BUILD
          
          // This function takes a static framework that is already linked to a different target in the project and links it to the specified target
          public static void InsertStaticFrameworkIntoTargetBuildPhaseFrameworks(string staticFrameworkName, string frameworkGuid, string target, ref string contents, PBXProject project) {
@@ -152,7 +152,7 @@
             //add the framework as an additional object in PBXBuildFile objects
             contents = contents.Replace("; fileRef = " + fileRef + " /* " + staticFrameworkName + ".a */; };", "; fileRef = " + fileRef + " /* " + staticFrameworkName + ".a */; };\n\t\t" + frameworkGuid + " /* " + staticFrameworkName + ".a in Frameworks */ = {isa = PBXBuildFile; fileRef = " + fileRef + " /* " + staticFrameworkName + ".a */; };");
 
-            //fild the build phase ID number
+            //find the build phase ID number
             string targetBuildPhaseId = project.GetFrameworksBuildPhaseByTarget(target);
             string[] components = contents.Split(new string[] { targetBuildPhaseId + " /* Frameworks */ = {\n\t\t\tisa = PBXFrameworksBuildPhase;\n\t\t\tbuildActionMask = " }, StringSplitOptions.None);
 

